### DISCUSSION TITLE
HitHub SOC2 Report

### DISCUSSION THREAD

**Discussion by romhub** [community/community#44461] [Enterprise]
Hello Team,

Could you please provide the source where or how the GitHub SOC2 report can be obtained?

Best regards,
Roman

**Comment by queenofcorgis** [comment]
Hey there - have you checked out this [doc](https://docs.github.com/en/enterprise-cloud@latest/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/accessing-compliance-reports-for-your-organization)? 

Hope that helps!

**Reply by dwyman-impact** [reply]
I, too, am trying to get a SOC 2 or ISO 27001 certificate. I currently use the Free plan and will be upgrading in the future but can't do it yet. Any way to find some sort of report or statement on Github's security posture for Free plan users? The link above only works if you're on the Enterprise plan.
